### PR TITLE
refactor: get_spawn_location

### PR DESCRIPTION
**Pull request by ghost**
Untested.. This should help shorter/longer models from being too far/close away to ped due to a static value being the offset before. also added some good math functions.

### CODE REVIEW

File: src/util/math.hpp [hunk 1]
```diff
@@ -23,6 +23,39 @@ namespace big::math
...
+	inline Vector3 multiply_v3(Vector3 vector, float x) {
+		Vector3 result;
+		result.x = vector.x;
+		result.y = vector.y;
+		result.z = vector.z;
+		result.x *= x;
+		result.y *= x;
+		result.z *= x;
+		return result;
+	}
```
Comment (Yimura) [inline src/util/math.hpp]: These can be moved to the rage::scrVector class as operator overloads.

And please have a look at the CONTRIBUTING.md file, your arguments are in camelCase and should be in snake_case.

**Review by Yimura — CHANGES_REQUESTED** [review]

**Review by Yimura — CHANGES_REQUESTED** [review]
Also run `clang-format` on your code.

**Comment by Rxann** [comment]
I don't believe he can format his code, as it is coded using the github online IDE.

**Comment by ghost** [comment]
> I don't believe he can format his code, as it is coded using the github online IDE. 

yuh I was too lazy to clone

**Comment by Rxann** [comment]
> > I don't believe he can format his code, as it is coded using the github online IDE.
> 
> yuh I was too lazy to clone

But you can't test the code to see if it even compiles......

**Comment by ghost** [comment]
Removed extra math, used overloads. Fixed formatting. Tested the artifact and seems to work fine. Should be good now...

**Review by Yimura — APPROVED** [review]
Could you code this on your local machine next time? Coding on Github.com directly is error prone and doesn't offer all the tooling you have locally (like clang-format, compiling the code, intellisense).

**Pull request merged**